setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
What's the benefit to making this pip-installable, I would've thought more likely to clone the repo and submit like that?

Reply:
We'd need to clone the repo within the batch command and set PYTHONPATH. Can do that of course, just an alternative approach